src/bootman/sysconfig.c

```diff
@@ -180,15 +182,13 @@ SystemConfig *cbm_inspect_root(const char *path, bool image_mode)
                         LOG_INFO("Fully resolved boot device: %s", rel);
                 }
                 c->wanted_boot_mask |= BOOTLOADER_CAP_GPT;
-        }
 
-        /* determine fstype of the boot_device */
-        if (c->boot_device) {
+                /* determine fstype of the boot_device */
                 c->wanted_boot_mask |= cbm_get_fstype(c->boot_device);
         }
 
         c->root_device = cbm_probe_path(realp);
-
+        
```

Comment:
Looks like you inserted a tab here by accident.

Reply:
done